javascript/game.js

```diff
@@ -1,117 +1,69 @@
+$(document).ready(function () {
+	var orange = $('#orange');
```

Comment:
Instead of setting the fruit variables to numbers, I made them jQuery wrapped representations of the image DOM nodes. This way you can store their value on through through the jQuery `.data` method.